test-fixture.html

```diff
@@ -313,9 +315,20 @@
     });
 
   try {
-    document.registerElement('test-fixture', {
-      prototype: TestFixturePrototype
-    });
+    if (window.customElements) {
+      function TestFixture() {
+        if (window.Reflect) {
+          return Reflect.construct(HTMLElement, [], TestFixture)
```

Comment:
Will fix. Is the `self` in your last example necessary? Seems like I could just return on that line.